k8s.io/configmap-nginx.yaml

```diff
@@ -287,6 +287,12 @@ data:
 
         location / {
           rewrite ^/.*$  https://prow.k8s.io/tide redirect;
+
+      server {
+        server_name yt.k8s.io youtube.k8s.io youtube.kubernetes.io yt.kubernetes.io;
```

Comment:
We need to add these to the certificate, too.  Followup is OK.

Reply:
pointer: https://github.com/kubernetes/k8s.io/blob/master/k8s.io/certificate-prod.yaml